src/components/fx/helpers.js

```diff
@@ -138,6 +138,11 @@ exports.makeEventData = function(pt, trace, cd) {
         if('yVal' in pt) out.y = pt.yVal;
         else if('y' in pt) out.y = pt.y;
 
+        if ('x0' in pt) out.x0 = pt.x0;
+        if ('x1' in pt) out.x1 = pt.x1;
+        if ('y0' in pt) out.y0 = pt.y0;
+        if ('y1' in pt) out.y1 = pt.y1;
```

Comment:
🌴 there's 7 or 8 of these... could be something like `copyCoords(pt, out)` - could even include the `xa/ya`-> `xaxis/yaxis` part, AFAICT only `scattercarpet` is missing that one, but I bet it wouldn't even hurt to include that.

Reply:
Is that information already accessible through `plotly_hover`? If so, can't we just modify `dcc.Graph` instead of making this PR?